Tests/MapboxCoreNavigationTests/RouteControllerTests.swift

```diff
@@ -162,3 +194,18 @@ extension RouteControllerTests: RouterDataSource {
         return NavigationLocationManager.self
     }
 }
+
+class AlternativeRoutesObserver: NavigatorAlternativesStoreObserver {
+    var onDidReportAlternatives: ((IndexSet, [AlternativeRoute]) -> Void)? = nil
+    var onDidFailToReportAlternatives: ((AlternativeRouteError) -> Void)? = nil
+    
+    func alternativesStore(_ store: NavigatorAlternativesStore, didReportNewAlternatives newAlternatives: IndexSet, removedAlternatives: [AlternativeRoute]) {
+        onDidReportAlternatives?(newAlternatives, removedAlternatives)
+    }
+    
```

Comment:
NIT: It seems that you don't have whitespace trimming enabled.
<img width="364" alt="Screen Shot 2022-04-26 at 16 03 58" src="https://user-images.githubusercontent.com/443972/165306096-4a8ec18c-10d8-4657-979a-b7f603c7aed7.png">

```suggestion
    }

```